packages/git-extractor/src/routes/github/api.ts

```diff
@@ -971,3 +975,26 @@ export async function downloadZip(
 
   return loadedZip;
 }
+
+export async function checkRemainingRateLimit(
```

Comment:
We can check GitHub's rate limit API as much as we want to see how many tries we have left

Reply:
**nitpick (non-blocking):** It may be a good call to add a comment here letting others know that this doesn't count against our rate-limit quota - something I was personally wondering.